packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -402,12 +422,15 @@ export const prepareSettler = (
           })();
           if (!dest) return;
 
-          const txfrV = E(settlementAccount).transfer(
-            dest,
-            AmountMath.make(USDC, fullValue),
-            { forwardOpts: { intermediateRecipient } },
-          );
-          void vowTools.watch(txfrV, this.facets.transferHandler, txHash);
+          const { reference } = parseAccountId(dest);
+          const amt = AmountMath.make(USDC, fullValue);
+          const txfrOrSendV =
```

Comment:
the `x` suggests "transaction" or some word with a "k" sound.

It's also not saving much. Consider `transferOrSendV`